static/js/custom.js

```diff
@@ -200,7 +199,7 @@ function updatePlaylist() {
                 200: displayPlaylist,
                 204: function () {
                     playlist_loading.hide();
-                    playlist_empty.show();
+                    playlist_empty.removeClass('d-none');
```

Comment:
This is more like a choice between jquery and bootstrap, and from your code, you are the bootstrap guy :)

Reply:
The thinking here is to prepare for a future without jQuery, too bad Bootstrap requires it.